pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1801,6 +1819,20 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 		case "debug.default.remote.loglevel":
 			newGlobalConfig.DefaultRemoteLogLevel = item.Value
 
+		case "storage.dom0MinDiskUsagePercent":
```

Comment:
camelCaseIsForProgrammersNotForUsers

Reply:
Fixed it..Changed it to: 
     storage.dom0.disk.minusage.percent